### DISCUSSION TITLE
[mesheryctl] Rename ~/.meshery/config.yaml to ~/.meshery/config

### DISCUSSION THREAD

**Discussion by leecalcote** [meshery/meshery#3063] [General]
We refer to the `config.yaml` file as meshconfig akin to kubeconfig. Kubernetes stores the kubeconfig file at `~/.kube/config` with not filename extension. 

To reinforce this affiliation and mental model for users and for contributors, I propose that `config.yaml` be renamed to `config`.

**Comment by pottekkat** [comment]
Since creation of config files are handled by Viper, we need to check if this is feasible...

**Reply by pottekkat** [reply]
I think it should, and I think that this is a good change 👍🏼 

See- https://github.com/spf13/viper#reading-config-files
